storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.7.rst

```diff
@@ -0,0 +1,60 @@
+.. _rn.8-0-7:
+
+================================================================================
+|pxb.name| |release|
+================================================================================
+
+|percona| is glad to announce the release of |pxb.name| |release| on |date|.
+Downloads are available from our `download site
+<https://www.percona.com/downloads/Percona-XtraBackup-LATEST/>`_ and from
+:ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+|pxb.name| enables MySQL backups without blocking user queries, making it ideal
+for companies with large data sets and mission-critical applications that cannot
+tolerate long periods of downtime. Offered free as an open source solution, it
+drives down backup costs while providing unique features for |MySQL| backups.
+
+In release |release|, |pxb.name| introduces the encryption of the system
+tablespace. Also, this new release supports the ``lz4`` compression algorithm
+directly which improves the speed of compression. Use `--compress=lz4` to use
```

Comment:
could be better to say that compressed backups can now use lz4 compression algorithm in addition to quicklz